src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1241,6 +1241,53 @@ internal bool ShouldEmitNullableContextValue(out byte value)
             return value != containingValue;
         }
 
+        /// <summary>
+        /// True if the symbol is declared outside of the scope of the containing
+        /// symbol
+        /// </summary>
+        internal static bool IsCaptured(Symbol variable, SourceMethodSymbol containingSymbol)
```

Comment:
>static [](start = 17, length = 6)

Should this be an instance method on `variable` or `SourceMethodSymbol`?

Reply:
It's not very commonly used, so I thought it would be better to avoid polluting intellisense